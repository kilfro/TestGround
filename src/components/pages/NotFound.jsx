import React from 'react';
import '../../styles/component/pages/notfound.css';
import {Button} from "@material-ui/core";

const NotFound = () => {
    return (
        <div className={'not-found-page'}>
            <div className="code-404">404</div>
            <div>Похоже страницы, которую Вы ищете, не существует</div>
            <Button href={'/'} size={'large'} variant={'outlined'}>Вернуться домой</Button>
        </div>
    )
};

export default NotFound;