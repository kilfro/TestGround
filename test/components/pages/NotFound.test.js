import React from 'react';
import NotFound from "../../../src/components/pages/NotFound";

describe('Not found page', () => {
    it('should render correctly', () => {
        const page = shallow(<NotFound/>);
        expect(page).toMatchSnapshot();
    });
});