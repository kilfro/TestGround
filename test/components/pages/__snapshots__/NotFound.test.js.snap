// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page should render correctly 1`] = `
<div
  className="not-found-page"
>
  <div
    className="code-404"
  >
    404
  </div>
  <div>
    Похоже страницы, которую Вы ищете, не существует
  </div>
  <WithStyles(ForwardRef(Button))
    href="/"
    size="large"
    variant="outlined"
  >
    Вернуться домой
  </WithStyles(ForwardRef(Button))>
</div>
`;
